pkg/controllers/management/multiclusterapp/handler_multiclusterapp.go

```diff
@@ -163,37 +182,6 @@ func (mc *MCAppController) sync(key string, mcapp *v3.MultiClusterApp) (runtime.
 				}
 			}
 		}
-
-		for _, p := range mcapp.Spec.Targets {
-			if p.ProjectName == "" {
-				continue
-			}
-			split := strings.SplitN(p.ProjectName, ":", 2)
-			if len(split) != 2 {
-				return nil, fmt.Errorf("invalid project name")
-			}
-			projectName := split[1]
-			// check if these prtbs already exist
-			_, err := mc.prtbLister.Get(projectName, prtbName)
-			if err != nil {
-				if apierrors.IsNotFound(err) {
-					_, err := mc.prtbs.Create(&v3.ProjectRoleTemplateBinding{
-						ObjectMeta: v1.ObjectMeta{
-							Name:      prtbName,
-							Namespace: projectName,
-						},
-						UserName:         systemUser.Name,
-						RoleTemplateName: r,
-						ProjectName:      p.ProjectName,
-					})
-					if err != nil && !apierrors.IsAlreadyExists(err) {
-						return nil, err
-					}
-				} else {
-					return nil, err
-				}
-			}
```

Comment:
What is this change that we are deleting?

Reply:
Oh, this is the PRTB generation logic. On Friday before merging the original PRs we realized we needed CRTB generation too. So I added a same for loop for PRTB and CRTB, and forgot to delete this original for loop.
This is where it's handled now https://github.com/rancher/rancher/pull/17978/files#diff-2370d970aad60679ffe6572e455106ceR122

And what I'm deleting is just the duplicate code @prachidamle